frontend/src/components/VaultDetails.vue

```diff
@@ -8,12 +8,20 @@
   </div>
 
   <div v-else class="pb-16 space-y-6">
+    <div v-if="vault?.archived" class="rounded-md bg-yellow-50 p-4">
+      <div class="flex">
+        <div class="flex-shrink-0">
+          <ExclamationTriangleIcon class="h-5 w-5 text-yellow-400" aria-hidden="true" />
+        </div>
+        <h4 class="ml-3 text-sm font-medium text-yellow-800">This vault is archived</h4>
+      </div>
+    </div>
     <div>
       <h3 class="font-medium text-gray-900">{{ t('vaultDetails.description.header') }}</h3>
       <div class="mt-2 flex items-center justify-between">
         <p v-if="vault != null && vault.description.length > 0" class="text-sm text-gray-500">{{ vault.description }}</p>
         <p v-else class="text-sm text-gray-500 italic">{{ t('vaultDetails.description.empty') }}</p>
-        <!-- TODO: add rest API to change vault metadata in backend
+        <!-- TODO: implement calls to change metadata (see backend.ts -> VaultService - update()) 
```

Comment:
Done?

Reply:
Nope, since the actual calls in `VaultDetails.vue` are not implemented. I wanted to seperate these matters.